get.sh

```diff
@@ -20,13 +20,20 @@ then
 	cd $1
 	mkdir openjdkbinary
 	cd openjdkbinary
-	download_url=`curl https://api.github.com/repos/AdoptOpenJDK/openjdk-releases/releases/latest | jq -r '.assets[] | select(.browser_download_url | contains("x64_Linux_jdk")) | {url: .browser_download_url} | select (.url | contains("tar.gz")) | .[]'`
-	wget "${download_url}"
-	#wget https://github.com/AdoptOpenJDK/openjdk-releases/releases/download/jdk8u152-b01/OpenJDK8_x64_Linux_jdk8u152-b01.tar.gz
-	jar_file_url=$download_url
-	jar_file_name=${jar_file_url##*/}
-	tar -zxvf $jar_file_name
-	#tar -zxvf OpenJDK8_x64_Linux_jdk8u152-b01.tar.gz
+	case $(uname) in
+    *CYGWIN*)
+		wget https://github.com/AdoptOpenJDK/openjdk-releases/releases/download/jdk8u152-b04/OpenJDK8_x64_Win_jdk8u152-b04.zip
+		unzip OpenJDK8_x64_Win_jdk8u152-b04.zip
+		;;
```

Comment:
This will be enhanced depending on how we would like to run the tests(e.g. nightly only run latest? developer run specific version? What is the available gitHub APIs...). Currently we'd like to enable the build first as it seems build on windows have more issues than on Linux.